apps/backpack/views.py

```diff
@@ -305,9 +303,6 @@ def pdf(request, *args, **kwargs):
     slug = kwargs["slug"]
     try:
         badgeinstance = BadgeInstance.objects.get(entity_id=slug)
-        # TODO: Check other recipient types 
```

Comment:
Does this maybe belong to another ticket?

Reply:
I think it`s not part of the PR anymore? Or did i overlook something